app/helpers/application_helper.rb

```diff
@@ -23,4 +23,8 @@ def status_label(status)
     end
     "<div style='white-space: nowrap;'><span class='label #{labelclass}'>#{label}</span></div>".html_safe
   end
+  
+  def support_url
+    ENV['OOD_DASHBOARD_SUPPORT_URL'] || "https://www.osc.edu/contact/supercomputing_support"
```

Comment:
Remove `|| "https://www.osc.edu/contact/supercomputing_support"` because this is not supposed to contain OSC specific references.

Reply:
Note if we have the same thing in another app, open an issue in that repo stating that default value of support_url should not be a link to the OSC website.